content/post/29-js-ipfs-pubsub.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2017-07-13
+url: 29-js-ipfs-pubsub
+title: Decentralized pubsub on the browser using js-ipfs
+author: Pedro Teixeira
+---
+
+Recently, js-IPFS introduced experimental support for pub-sub.
```

Comment:
Well, it was like September last year 😅

Use `[js-ipfs](https://github.com/ipfs/js-ipfs)` instead of js-IPFS
Use `PubSub` instead of `pub-sub`

What about:

`[js-ipfs](https://github.com/ipfs/js-ipfs) has support for PubSub behind a experimental flag.`